<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTab</class>
 <widget class="QWidget" name="FileTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Tab</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QSplitter" name="preprocess_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="file_list_group">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>List of files:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CustomListWidget" name="file_list_widget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QPushButton" name="show_preprocessing_options">
           <property name="text">
            <string>Show preprocessing options</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="add_file_button">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Add files...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="show_full_path_check_box">
           <property name="text">
            <string>Show full path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="remove_checked_file_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove checked</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="preprocess_group">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Preprocessing</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="conversion_group">
         <property name="title">
          <string>Conversion to XYZR format</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="auto_format_radio_button">
            <property name="text">
             <string>Determine format automatically</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">format_button_group</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="other_format_radio_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Use the following file format:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">format_button_group</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="custom_format_radio_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Custom file format</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">format_button_group</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="Line" name="static2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="element_table">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>57</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Z</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Symbol</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>VdW radius</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Colour</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="new_element_button">
            <property name="text">
             <string>Add another element</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout5">
         <item>
          <widget class="QCheckBox" name="delete_loaded_check_box">
           <property name="text">
            <string>Delete loaded molecules</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_molecules">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Load molecules</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header>ui/widgets/custom_list_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="format_button_group"/>
 </buttongroups>
</ui>
